.gitignore

```diff
@@ -2,5 +2,7 @@
 /vendor/
 /composer.lock
 /.phpsu/
+/tests/.phpsu
 /tests/test-results/
 /phpsu-config.php
+/.phpunit-watcher-cache.php
```

Comment:
https://github.com/spatie/phpunit-watcher/commit/f8e360d9e4fe4280229f6d425f3bf02127a141fd

This dosn't need an exclude as it is only present if you got a very old version of phpunit-watcher.
How did you get an so old version?

Reply:
Testing with many versions